tideways/provision.sh

```diff
@@ -68,11 +71,19 @@ function restart_php() {
 
 function install_xhgui_frontend() {
     cp -f "${DIR}/nginx.conf" "/etc/nginx/custom-utilities/xhgui.conf"
+    declare -a packages=()
+    for version in "7.0" "7.1" "7.2" "7.3" "7.4"; do
+        if [[ $(command -v php$version) ]]; then
+            packages+=("php${version}-sqlite3")
+        fi
+    done
+    apt-get -y --allow-downgrades --allow-remove-essential --allow-change-held-packages -o Dpkg::Options::=--force-confdef -o Dpkg::Options::=--force-confnew install --fix-missing --fix-broken "${packages[@]}"
     if [[ ! -d "/srv/www/default/xhgui" ]]; then
         echo -e " * Git cloning xhgui from https://github.com/perftools/xhgui.git"
         cd /srv/www/default
         noroot git clone "https://github.com/perftools/xhgui.git" xhgui
         cd xhgui
+        composer remove alcaeus/mongo-php-adapter
```

Comment:
what's the reasoning behind this? Won't this change the lock file preventing further updates via `git`?

Reply:
Right now if mongo is not installed xhgui won't start but now is not using anymore mongo.

Reply:
I opened a ticket on XHgui https://github.com/perftools/xhgui/issues/364

Reply:
according to that ticket PDO support is incomplete, which means the other change to use a tagged release may be incompatible with the changes to move to sqlite